src/test/java_test/step1/StringTest.java

```diff
@@ -0,0 +1,54 @@
+package java_test.step1;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class StringTest {
+    @Test
+    void splitStringNumbers() {
+        String inputString = "1,2";
+        String[] splitedString = inputString.split(",");
```

Comment:
```suggestion
        final String[] actual = "1,2".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?